src/app/services/components/ExternalServiceDetails.vue

```diff
@@ -51,7 +47,6 @@ import { useRoute } from 'vue-router'
 import { ExternalService } from '@/types/index.d'
 import { stripTimes } from '@/utilities/helpers'
 import TagList from '@/app/common/TagList.vue'
-import EntityURLControl from '@/app/common/EntityURLControl.vue'
 import YamlView from '@/app/common/YamlView.vue'
 
 const route = useRoute()
```

Comment:
**Suggestion**: Please remove the no longer needed definition of `route` and its related import (neat that GitHub funnels linter warnings into this view).